metron-interface/metron-config/src/app/sensors/effects/sensors.effects.spec.ts

```diff
@@ -129,16 +129,18 @@ describe('sensor.effects.ts', () => {
     store.dispatch(new fromActions.ApplyChanges());
   });
 
-  it('Should show notification when operation SUCCESSFULL', () => {
-    store.dispatch(new fromActions.ApplyChanges());
-    expect(userNotificationSvc.showSuccessMessage).toHaveBeenCalled();
-  });
+  // Removing these tests until we rewrite this test suite to use a mock store and mock actions
```

Comment:
Does this have tasks added under the Jira for feature branch completion? Please link to this PR.

Reply:
It does now: https://issues.apache.org/jira/browse/METRON-2277

Thanks for pointing this out, @mmiklavc.